src/utils/displayFunctions.ts

```diff
@@ -21,6 +21,25 @@ const getLogoForBrandPetname = (brandPetname: string) => {
   }
 };
 
+export const addCommas = (stringifiedValue: string) => {
```

Comment:
this is locale-specific and will be wrong in some countries. fortunately there are platform facilities for it. please use `Intl.NumberFormat` instead (within the display functions since it's part of the stringification)

Reply:
We lose precision with `Number` but maybe that's okay for displaying prices because of practical limits. For amounts it could be a problem if we ever deal with an asset with a very large total supply, and I was just assuming it was okay to use a standard price format because we're dealing with USD. Not sure what's better.

Reply:
Actually in practice it seems the precision issue is a lot easier to hit for decimals:
```
new Intl.NumberFormat().format(Number('1.232421'))
'1.232'
new Intl.NumberFormat().format(Number('1.232421234'))
'1.232'
new Intl.NumberFormat().format(Number('100000000011231.23242'))
'100,000,000,011,231.23'
new Intl.NumberFormat().format(Number('100000000000000023452542523452453253253'))
'100,000,000,000,000,020,000,000,000,000,000,000,000'
```

Reply:
> standard price format because we're dealing with USD

USD is what the number is counting, but numbers themselves are locale-specific. For example, in Argentina it would be "$ 1.234,56".

> precision issue

Hmm. I hadn't considered that. Try https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString

Reply:
I just figured it might not be too shocking to see USD in US locale, just not the best UX.

`BigInt.toLocaleString()` doesn't work because it doesn't know about decimal places. I realized the precision isn't really an issue for cases where we're showing 0 or 2 decimal places though, so just made it work for that case. PTAL

Reply:
lgtm. some feedback there but for this thread: please delete `addCommas`

Reply:
Oops, accidently came back during rebase. removed